app/src/routes/browse.js

```diff
@@ -0,0 +1,10 @@
+import { Link } from "react-router-dom";
+
+export default function Browse() {
```

Comment:
There should be one more page after this. I was thinking we call it Confirm? Or maybe ConfirmJob to be more specific